addons/mail/tests/test_mail_composer.py

```diff
@@ -283,4 +283,6 @@ def test_mail_composer_test_tour(self):
             )
         message = self._new_msgs.filtered(lambda message: message.author_id == self.user_employee.partner_id)
         self.assertEqual(len(message), 1)
+        self.assertEqual(message.attachment_ids[0].raw, b'hi there')
+        self.assertEqual(message.attachment_ids[1].raw, b'hello, world')
```

Comment:
Would probably sort results (i.e. compare all raw, sorted), as we don't want to have an issue due to some ordering
```suggestion
        self.assertEqual(
            sorted(message.attachment_ids.mapped('raw')),
            sorted([b'hello, world', b'hi there']))
```